test/system/030-run.bats

```diff
@@ -222,6 +222,10 @@ echo $rand        |   0 | $rand
 
     # All OK. Kill container.
     run_podman rm -f $cid
+    run stat $cidfile
+    if [ "$status" -ne 1 ]; then
```

Comment:
I think `if [[ -e $cidfile ]]` is a little easier to read, but not worth repushing for. LGTM, thank you!

Reply:
Will change in case I need to repush, thanks!